packages-native/barcode-scanner/package.json

```diff
@@ -1,28 +1,32 @@
 {
-    "name": "barcode-scanner",
-    "widgetName": "BarcodeScanner",
-    "version": "0.1.3",
-    "license": "Apache-2.0",
-    "repository": {
-        "type": "git",
-        "url": "https://github.com/mendix/widgets-resources.git"
-    },
-    "config": {
-        "widgetPath": "../test-project/mxproject/widgets",
-        "projectPath": "../test-project/mxproject"
-    },
-    "packagePath": "com.mendix.widget.native",
-    "scripts": {
-        "start": "..\"/../node_modules/.bin/pluggable-widgets-tools\" start:ts:native -- --subprojectPath packages-native/barcode-scanner",
-        "build": "..\"/../node_modules/.bin/pluggable-widgets-tools\" build:ts:native --subprojectPath packages-native/barcode-scanner",
-        "release": "..\"/../node_modules/.bin/pluggable-widgets-tools\" release:ts:native --subprojectPath packages-native/barcode-scanner",
-        "lint": "..\"/../node_modules/.bin/eslint\" --config ../../.eslintrc.js --ext .jsx,.js,.ts,.tsx src/",
-        "test": "..\"/../node_modules/.bin/jest\" --config ../../scripts/test/jest.native.config.js",
-        "version": "npm run release",
-        "postversion": "..\"/../node_modules/.bin/ts-node\" ../../scripts/release/UpdateGithubRelease.ts --project ../../scripts/tsconfig.json"
-    },
-    "dependencies": {
-        "@native-mobile-resources/util-widgets": "^1.0.0",
-        "@widgets-resources/piw-utils": "^1.0.0"
-    }
-}
+  "name": "barcode-scanner",
+  "widgetName": "BarcodeScanner",
+  "version": "0.1.3",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mendix/widgets-resources.git"
+  },
+  "config": {
+    "widgetPath": "../test-project/mxproject/widgets",
+    "projectPath": "../test-project/mxproject"
+  },
+  "packagePath": "com.mendix.widget.native",
+  "scripts": {
+    "start": "pluggable-widgets-tools start:ts:native -- --subprojectPath packages-native/barcode-scanner",
+    "build": "pluggable-widgets-tools build:ts:native --subprojectPath packages-native/barcode-scanner",
+    "release": "pluggable-widgets-tools release:ts:native --subprojectPath packages-native/barcode-scanner",
+    "lint": "eslint --config ../../.eslintrc.js --ext .jsx,.js,.ts,.tsx src/",
+    "test": "jest --config ../../scripts/test/jest.native.config.js",
+    "version": "npm run release",
+    "postversion": "ts-node ../../scripts/release/UpdateGithubRelease.ts --project ../../scripts/tsconfig.json"
+  },
+  "dependencies": {
+    "@native-mobile-resources/util-widgets": "^1.0.0",
+    "@widgets-resources/piw-utils": "^1.0.0",
+    "react-native-camera": "3.19.2"
```

Comment:
Isnt react-native-camera already inside the tools?

Reply:
No, no "provided by client" packages are dependencies of tools.